tensorflow/compiler/mlir/tensorflow/ir/tf_generated_ops.td

```diff
@@ -1815,7 +1815,8 @@ def TF_ConfigureTPUEmbeddingOp : TF_Op<"ConfigureTPUEmbedding", []> {
   let results = (outs);
 }
 
-def TF_ConjOp : TF_Op<"Conj", [NoSideEffect, SameOperandsAndResultType]> {
+def TF_ConjOp : TF_Op<"Conj",
+                [Involution, NoSideEffect, SameOperandsAndResultType,]> {
```

Comment:
Can the traits be on the same line as the op definition (like prior to your change)? This file is mostly generated and that is the expected format. Also can the trailing comma in traits be removed?